svg/painting/marker-007.svg

```diff
@@ -0,0 +1,23 @@
+<svg width="100%" height="100%" viewBox="0 0 480 360"
```

Comment:
Yes, Gecko doesn't seem to conform to the spec on that one. I can't find a bug in bugzilla, but I'll raise one once the test is reviewed and lands.
Here's the SVG 1.1 SE test suite link for reference: https://www.w3.org/Graphics/SVG/Test/20110816/harness/htmlObjectApproved/painting-marker-07-f.html